src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
```

Comment:
The opening `<html>` tag is self-closing, which is incorrect. HTML tags should not be self-closed. Please change `<html />` to `<html>`.